app/models/application_record.rb

```diff
@@ -1,3 +1,6 @@
 class ApplicationRecord < ActiveRecord::Base
   self.abstract_class = true
+  EMAIL_REGEXP = /\A[\w+\-.]+@[a-z\d\-]+(\.[a-z\d\-]+)*\.[a-z]+\z/i
+  ALPHA_REGEXP = /\A[a-z A-Z]+\z/
```

Comment:
this regexx is not being used

Reply:
It will in future models validations. I have an model with CODE (PK) alphanumeric.

Reply:
Then please add it in the future when you add that code, let's make sure to only add code related with the current implementation, otherwise we lose the focus of what is being built at the moment

Reply:
Right!

Reply:
don't declare constants or variables until they are necessary

Reply:
I can add this after, to validation? There are other approach for this ?

Reply:
it seems that the best way is indeed using a regex